mobile/cmp/grid/impl/ColChooser.js

```diff
@@ -114,56 +111,58 @@ export class ColChooser extends Component {
                 button({
                     text: 'Save',
                     icon: Icon.check(),
-                    onClick: this.onOK
+                    onClick: () => {
+                        model.commit();
+                        model.close();
+                    }
                 })
             ]
         });
     }
+});
 
-    //------------------------
-    // Implementation
-    //------------------------
-    renderColumnList(columns, props = {}) {
-        const {placeholder, className, ...rest} = props;
-
+//------------------------
+// Implementation
+//------------------------
+const columnList = hoistCmp.factory({
+    render({cols, placeholder, className, ...props}, ref) {
```

Comment:
Where is this 'ref' used? There is a 'ref' passed as part of props, but not as a second argument